test/test_board.py

```diff
@@ -806,6 +806,39 @@ def test_initial_diamond_position(self, row_random=4, expected_result=1):
             .diamond, expected_result
         )
 
+    def test_only_16_holes_have_to_be_put(self):
```

Comment:
Don't use values in the name of tests

Reply:
Ok, i'll change it to test_only_right_HOLE_QUANTITY_have_to_be_placed